### DISCUSSION TITLE
Add Delay between every retry

### DISCUSSION THREAD

**Discussion by engahmedtahaa** [jobrunr/jobrunr#144] [Q&A] [answered]
while we are set retries numbers on the job some times we faced table lock issues and job performance exceptions due to the time between every retry can we configure the duration or what is the best solution on this case?

**Comment by rdehuyss** [comment, marked as answer]
Have you seen the javadocs of the [RetryFilter](https://github.com/jobrunr/jobrunr/blob/master/core/src/main/java/org/jobrunr/jobs/filters/RetryFilter.java)?

That should explain everything.

**Reply by rdehuyss** [reply]
Hi @engahmedtahaa, the question you asked here keeps me lingering... You talk about table locks but as far as I know, JobRunr does not take any table locks.

Could you elaborate a bit on the exact problem you have/had? Why do you think there are table locks? And what database are you using?
